src/sv-pipeline/04_variant_resolution/scripts/process_posthoc_cpx_depth_regenotyping.sh

```diff
@@ -103,7 +103,7 @@ if ! [ -s ${INVCF} ]; then
   usage
   exit 0
 fi
-if [ $( file ${INVCF} | fgrep "gzip" | wc -l ) -lt 1 ]; then
+if [[ ${INVCF} != *.gz ]]; then
```

Comment:
I suppose this check can be "tricked" if one creates any file (compressed or uncompressed) and just adds `.gz` extension. Otherwise, I guess this check is good enough.

Reply:
In my opinion this is much more stable. When I updated our docker images, the output of `file` changed in a way that made the checks break for some files. If parsing the tool output causes breaks when you change Ubuntu versions, it seems better to me to just check the extension. Especially since this code will only be parsing output from previous scripts in our pipeline: if it breaks this way, it's our fault.